srv/modules/runners/replace.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=too-few-public-methods,modernize-parse-error
+"""
+Runner to remove a single osd
+"""
+
+from __future__ import absolute_import
+from __future__ import print_function
+import time
+import logging
+import os
+# pylint: disable=import-error,3rd-party-module-not-gated,redefined-builtin
+import salt.client
+import salt.runner
+
+log = logging.getLogger(__name__)
+
+
+def help_():
+    """
+    Usage
+    """
+    usage = ('salt-run replace.osd id [id ...][force=True]:\n\n'
```

Comment:
That should reflect the _supported_ args of the respective module '['force', 'timeout', 'delay']'

Reply:
Hmm, the timeout and delay are passed to the osd.remove module, so the defaults aren't set here.  Also, only using those two in smoketests currently.  I will have to think about what to put here.